content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -241,6 +241,12 @@ kubectl drain <node-to-drain> --ignore-daemonsets
 
 ### Upgrade kubelet and kubectl
 
+{{< note >}}
+On Linux nodes, the kubelet defaults to supporting only cgroups v2.
+For Kubernetes {{< skew currentVersion >}} the `FailCgroupV1` kubelet configuration option is set to true by default.
+Refer to the [Kubernetes cgroup v1 deprecation documentation](/docs/concepts/architecture/cgroups/#deprecation-of-cgroup-v1) for more information.
+{{</ note >}}
+
```

Comment:
as i mentioned above, i think we should only talk about this in the upgrading-linux-nodes page

Reply:
Miss that. Updated.

Reply:
> I would mention cgroupv2 in https://kubernetes.io/docs/tasks/administer-cluster/cluster-upgrade/ too
> 
> kubeadm is only one of several viable ways to manage nodes and that allows for in-place upgrades.

Do you mean adding a same note in https://kubernetes.io/docs/tasks/administer-cluster/cluster-upgrade/#upgrade-kubeadm section? Or in https://kubernetes.io/docs/tasks/administer-cluster/cluster-upgrade/#before-you-begin?